### PR TITLE
http: Fixes for Gin http receiver sample

### PR DESCRIPTION
**Pull request by dgeorgievski**
Fixes for prematurely closed #842 PR. Applied all suggested code and doc changes.

It is worth mentioning the code and instructions come from a real use case of processing Tekton events:
https://tekton.dev/docs/pipelines/events/

### CODE REVIEW

**Comment by duglin** [comment]
don't forget to sign your commit(s)

**Comment by duglin** [comment]
ping @embano1

**Comment by dgeorgievski** [comment]
> don't forget to sign your commit(s)

Signed off. Sorry, I forgot.

**Comment by duglin** [comment]
thanks.
ping @embano1 and @lionelvillard for review

File: samples/http/receiver-gin/README.md [hunk 1]
```diff
@@ -18,11 +17,11 @@ Run the app
  go run main.go
 ```
 
-Test a CloudEvent
+Send a CloudEvent
 ```shell
 curl -v \
  -H "Ce-Id: e7d95c20-6eb4-4614-946d-27b0ce41c7ff" \
- -H "Ce-Source: /apis///namespaces/dimitar//clone-build-n4qhgl" \
+ -H "Ce-Source: /apis/namespaces/dimitar/clone-build-n4qhgl" \
```
Comment (embano1) [inline samples/http/receiver-gin/README.md:24]: You need to change this in the output below too

**Review by embano1 — CHANGES_REQUESTED** [review]
One nit

File: samples/http/receiver-gin/README.md (hunk 1 above)
Comment (dgeorgievski) [reply samples/http/receiver-gin/README.md:24]: Source changed in the output as well.

**Comment by duglin** [comment]
LGTM

@embano1 and @lionelvillard FTM (for the merge)

**Comment by duglin** [comment]
ping @lionelvillard for final approval/merge

**Pull request merged**